components/operators-detail/documentation/documents-by-stacked-timeline.js

```diff
@@ -22,53 +27,39 @@ class DocumentsByTime extends React.Component {
     }
   }
 
+
   componentWillReceiveProps(nextProps) {
     if (nextProps.documentation.length !== this.props.documentation.length) {
       this.drawChart(nextProps.documentation);
     }
   }
 
-  shouldComponentUpdate() {
```

Comment:
Be careful, you are using d3. I think that you don't want to re-render the graph ever